Ports/libfftw3f/package.sh

```diff
@@ -8,3 +8,8 @@ files=(
     "http://fftw.org/fftw-${version}.tar.gz#56c932549852cddcfafdab3820b0200c7742675be92179e59e6215b340e26467"
 )
 workdir="fftw-${version}"
+
+post_install() {
+    # Some upstream libraries search for "fftw3" in pkgconfig, not "fftw3f".
+    ln -sf "fftw3f.pc" "${SERENITY_INSTALL_ROOT}/usr/local/lib/pkgconfig/fftw3.pc"
```

Comment:
These are not the same library. One is configured for single precision, the other defaults to double precision.

Reply:
Huh, missed that. I wonder why it managed to link fine.